autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ClusterStatusSyncHandler.java

```diff
@@ -42,17 +42,18 @@ public void onApplicationEvent(ClusterStatusSyncEvent event) {
 
         StackStatusV4Response statusResponse = cloudbreakCommunicator.getStackStatusByCrn(cluster.getStackCrn());
 
-        // TODO CB-14929: cluster availability checks needs to be based on the scaling mechanism, and the eventual status of
-        // a cluster which has STOPPED instances.
-//        boolean clusterAvailable = Optional.ofNullable(statusResponse.getStatus()).map(Status::isAvailable).orElse(false)
-//                && Optional.ofNullable(statusResponse.getClusterStatus()).map(Status::isAvailable).orElse(false);
+        boolean clusterAvailable;
+        boolean clusterNodesUnhealthy = false;
+        if (cluster.isStopStartScalingEnabled()) {
```

Comment:
Since this would require changes coming in from #12026 , will examine the possibilities more carefully, and create a separate PR as part of [CB-15141](https://jira.cloudera.com/browse/CB-15141)